package.json

```diff
@@ -2,11 +2,14 @@
   "private": true,
   "scripts": {
     "build": "vitruvius build --src src --dest lib --ignore-packages \"**/integration-tests/**\"",
-    "clean": "rimraf ./packages/*/lib",
+    "prebuild": "rimraf ./packages/*/lib",
     "lint": "eslint . --cache",
-    "postinstall": "yarn run clean --silent && yarn run build --silent",
+    "postinstall": "yarn build --silent",
     "precommit": "lint-staged",
-    "test": "jest",
+    "setup": "lerna clean --yes && lerna bootstrap",
+    "storybook": "lerna run --scope=react-percy-storybook-integration-tests storybook --stream",
+    "test": "jest --ci --no-cache --coverage",
```

Comment:
Given how rarely we change the babel config relative to how often we run tests, I think we ought to let Jest cache to keep things fast.